views/signup.erb

```diff
@@ -0,0 +1,213 @@
+			</a>
+		</li>
+	</ol>
+</nav>
+
+<div class="container">
+	<div class="grid-row">
+		<div class="column-full filter-fields">
+			<div class="back">
+				<a href="/" class="link-back step-back">Back</a>
+			</div>
+		</div>
+		<div class="column-two-thirds">
+			<h1 class="heading-large">Request a Gov.UK PaaS account</h1>
+			<% if @errors && @errors.size > 0 %>
+				<div class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+					<h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">There were problems submitting your request</h2>
+					<ul class="error-summary-list">
+						<% @errors.each do |field, err| %>
+							<li class="err"><a href="#" class="focus-field" data-field="<%= field.to_s %>"><%= err %></a></li>
+						<% end %>
+					</ul>
+				</div>
+			<% end %>
+			<form action="/signup" method="post">
+
+				<div class="step step--join-organisation">
+					<p>If your team is already using GOV.UK PaaS and you would like a user account, ask your Org Manager to contact us and make a request at:<br><a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk">gov-uk-paas-support@digital.cabinet-office.gov.uk</a></p>
+					<p>Org Managers administer the account and assign roles and permissions.</p>
+					<p>Users with non-government email addresses can be added to an account at the request of an Org Manager.</p>
+					<div class="form-group">
+						<a class="button button-start" href="/contact-us">Contact Support</a> </p>
+					</div>
+				</div>
+
+				<div class="step step--start step--all">
+					<p>Service teams on GOV.UK PaaS own and use a development account called an <em>organisation</em>.</p>
+					<div class="form-group">
+						<fieldset>
+							<legend>
+								<span class="form-label-bold">Do you want to create an Organisation?</span>
+								<span class="form-hint">An organisation typically represents the real-world organisation, department or team you work for.</span>
+							</legend>
+							<div class="multiple-choice">
+								<input id="choose-create" type="radio" name="step" value="step--about-you" checked="checked">
+								<label for="choose-create">Yes, I'm new to PaaS and want to create an organisation</label>
+
+							</div>
+							<div class="multiple-choice">
+								<input id="choose-join" type="radio" name="step" value="step--join-organisation">
+								<label for="choose-join">No, I'd like to join an existing organisation</label>
+							</div>
+						</fieldset>
+					</div>
+					<div class="form-group">
+						<button class="button step-button step--1-button" style="display:none;" disabled="disabled">Continue</button>
+					</div>
+				</div>
+
+				<div class="step step--about-you step--all">
+					<h2 class="heading-medium">Your contact details</h2>
+					<div class="form-group <%= 'form-group-error' if @errors.has_key? :person_name %> form-group--person_name">
+						<label class="form-label" for="person_name">
```

Comment:
partial/helpers were created to render the form fields for the contact-us form.... these should probably be used on the signup form too for consistancy and to simplify the template